resources/sdk/purecloudgo/extensions/config.toml

```diff
@@ -0,0 +1,6 @@
+# Just leaving this here for now
+# Logging
+log_level="trace"
+log_format="text"
+log_to_console="true"
+log_file_path="/Users/ronanwatkins/genesys_src/repos/platform-client-sdk-common/output/purecloudgo/platformclientv2/log.log"
```

Comment:
Might want to change this to a Generic value rather then hardcoded to your path,

Reply:
That's from an earlier commit. All the fresh code is here: https://github.com/purecloudlabs/platform-client-sdk-common/pull/154/files